### ISSUE TITLE
ERL-264: Can't compile with --enable-m64-build and HIPE enabled

### ISSUE THREAD

**Issue by OTP-Maintainer** [erlang/otp#3532]
Original reporter: `dmilith`
Affected versions: `OTP-18.0`, `OTP-19.0`, `OTP-19.1`
Component: `compiler`
Migrated from: https://bugs.erlang.org/browse/ERL-264

---

```
Configure options used:



{noformat}
--enable-threads --with-dynamic-trace=dtrace --enable-fp-exceptions --enable-hipe --enable-kernel-poll --enable-smp-support --enable-sctp --with-ssl=${PREFIX} --disable-debug --enable-vm-probes --with-odbc=${PREFIX} --enable-native-libs --with-libatomic_ops=${PREFIX}
{noformat}




In short:

```
erl_prettypr.erl: internal error in native_compile;
        # crash reason: undef
```

- A lots of them.
Does exactly the same thing under Darwin 10.11, FreeBSD 11.. (both x86_64)
```
